rclcpp/include/rclcpp/wait_for_message.hpp

```diff
@@ -78,11 +79,13 @@ bool wait_for_message(
 /// Wait for the next incoming message.
 /**
  * Wait for the next incoming message to arrive on a specified topic before the specified timeout.
- *
- * \param[out] out is the message to be filled when a new message is arriving.
+ * Specify the QoS settings for the subscription to control how messages are received.
```

Comment:
i would remove this comment, it is pretty much the duplication of `qos` parameter explanation in the main body.